systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Signup" %}{% endblock %}
```

Comment:
Once you change the base template, you will see head title block is named differently. Follow same pattern as in "account/login.html".